clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -8160,6 +8160,43 @@ void StreamAPICallRule::runRule(const MatchFinder::MatchResult &Result) {
     EA.applyAllSubExprRepl();
     return;
   }
+  if (FuncName == "cudaStreamIsCapturing") {
+    if (!DpctGlobalInfo::useExtGraph()) {
+      report(CE->getBeginLoc(), Diagnostics::TRY_EXPERIMENTAL_FEATURE, false,
+             "cudaStreamIsCapturing", "--use-experimental-features=graph");
+      return;
+    }
+    std::string ReplStr;
+    std::string StreamName;
+    auto StmtStr0 = getStmtSpelling(CE->getArg(1));
+    // TODO: simplify expression
+    if (StmtStr0[0] == '&')
+      ReplStr = StmtStr0.substr(1);
+    else
+      ReplStr = "*(" + StmtStr0 + ")";
+    ReplStr += " = ";
+    bool IsDefaultStream = isDefaultStream(CE->getArg(0));
```

Comment:
It seems not need this variable, just put `isDefaultStream(CE->getArg(0))` in the if condition.